renaud.mk

```diff
@@ -1,4 +1,4 @@
 INSTANCE_ID = renaud
 VARS_FILE = vars_renaud.yaml
-
+PRINT_OUTPUT = /tmp
```

Comment:
shouldn't we include that param in geoportail_v3.mk rather than adding it in every make file?
in main.mk we can still oberride it with another dir if necessary